packages/net/tcp/connection.pony

```diff
@@ -1,3 +1,5 @@
+use "net"
+use "net/ip"
 use "collections"
 
```

Comment:
I think this file should be named `tcp_connection.pony`, because the principal type declared in the file is `TCPConnection`.